x-pack/plugins/threat_intelligence/cypress/e2e/cases.cy.ts

```diff
@@ -74,10 +70,14 @@ const deleteCase = () => {
 describe('Cases with invalid indicators', () => {
   before(() => {
     esArchiverLoad('threat_intelligence/invalid_indicators_data');
+    login();
```

Comment:
@patrykkopycinski  In this file we have 2 top-level `describe` sections ('Cases with invalid indicators' here and 'Cases interactions' a bit below).
Wouldn't we want the `login()` in the second one as well (looking at [this documentation](https://docs.cypress.io/api/commands/session#Where-to-call-cy-visit))

Reply:
that's correct, Thank you @PhilippeOberti

Reply:
@PhilippeOberti Thank you! Great catch! :)